packages/core/src/App.ts

```diff
@@ -116,6 +208,31 @@ export class App extends Module {
 		return this.modules;
 	}
 
+	public async getModuleById<M extends Module = Module>(moduleId: string, waitForStatus?: ModuleStatus): Promise<M> {
+		const statusOrders = [
```

Comment:
```suggestion
		const STATUS_ORDERS= [
```

If this is a constant, maybe better change the case here. Is this constant only relevant for this method? Should it be outside the function?

Reply:
good point, the array is relevant only for the function, I've renamed it for clarity :+1: